src/main/java/uk/gov/hmcts/reform/sscs/service/RoboticsService.java

```diff
@@ -50,7 +52,7 @@ public void sendCaseToRobotics(SscsCaseData caseData, Long caseId, String postco
     public void sendCaseToRobotics(SscsCaseData caseData, Long caseId, String postcode, byte[] pdf, Map<String, byte[]> additionalEvidence) {
         String venue = airLookupService.lookupAirVenueNameByPostCode(postcode);
 
-        JSONObject roboticsJson = createRobotics(RoboticsWrapper.builder().sscsCaseData(caseData)
```

Comment:
@chris-moreton , it's more thread-safe to return the json in this method rather than store it in a field to use it later on. I'd change it.

Reply:
@iguannago - Thanks - I would prefer to do it that way - my concerns were that I would be changing the interface which may affect other consumers of the library.